### ISSUE TITLE
Error: Cannot find module 'strip-ansi'

### ISSUE THREAD

**Issue by rajinder-yadav** [typicode/husky#136]
I am getting this error

```sh
> husky@0.13.4 uninstall /home/yadav/dev/projects/tscli/node_modules/husky

> node ./bin/uninstall.js

module.js:487
    throw err;
    ^

Error: Cannot find module 'strip-ansi'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/yadav/dev/projects/tscli/node_modules/chalk/index.js:4:17)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
npm WARN lifecycle husky@0.13.4~uninstall: continuing anyway husky@0.13.4 uninstall: `node ./bin/uninstall.js`
npm WARN lifecycle Exit status 1
npm WARN prefer global marked@0.3.6 should be installed with -g

> husky@0.13.4 install /home/yadav/dev/projects/tscli/node_modules/husky
> node ./bin/install.js
```

**Comment by rajinder-yadav** [comment]
I only see the error on a freshly installed project, if I rerun my project `npm run build` script it seems husky is happy.

**Comment by chrisui** [comment]
Looks similar to https://github.com/typicode/husky/issues/109

**Comment by typicode** [comment]
@rajinder-yadav thanks for the report.
FWIW `yarn` doesn't have this issue, but I'm working on a workaround in `next` branch for `npm`.

**Comment by typicode** [comment]
Just published a new beta version that should fix it.
To try it:

```sh
npm install husky@beta
```

Let me know if you still have the issue.

**Comment by typicode** [comment]
Hey, just published husky `v0.14`, it should fix this issue.
